types/marked/index.d.ts

```diff
@@ -137,8 +137,8 @@ declare namespace marked {
         codespan(code: string): string;
         br(): string;
         del(text: string): string;
-        link(href: string, title: string, text: string): string;
-        image(href: string, title: string, text: string): string;
+        link(href: string | null, title: string | null, text: string): string;
```

Comment:
I suppose without context, `title` could be any `falsy` value not just `null`. 

In any case I'm not very familiar with the library so I'll approve and merge this.